blogposts/2020/announcing-sourcegraph-3.17.md

```diff
@@ -0,0 +1,257 @@
+</style>
+
+We plan to continue on this path of performance improvements, and the next release will focus on processing multiple bundles concurrently in order to multiply the benefit of this release’s raw latency gains.
+
+For detailed technical information about recent performance boosts, [read the blog post](https://about.sourcegraph.com/blog/performance-improvements-in-precise-code-intel) on performance improvements for precise code intelligence.
+
+
+### Automatic precise code intelligence
+
+<div class="container my-4 video-embed embed-responsive embed-responsive-16by9 ">
+    <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/BHYka1CT700?autoplay=0&amp;cc_load_policy=0&amp;start=0&amp;end=0&amp;loop=0&amp;controls=1&amp;modestbranding=0&amp;rel=0" allowfullscreen="" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" frameborder="0"></iframe>
+</div>
+
+An experimental feature on [sourcegraph.com/search](https://sourcegraph.com/search) will automatically create an [LSIF index](https://docs.sourcegraph.com/user/code_intelligence/lsif) for eligible repositories based on their popularity, so that repository will have precise results on hover, definition, and reference operations. We are currently able to index Go repositories containing a go.mod file that does not require additional build steps and are working on expanding the set of eligible repositories to support additional languages and more sophisticated repository structures.
+
+To see this experiment in action, pick a Go repository hosted on GitHub that has yet to be discovered. Possibly something useful with a passionate cult-following. Then, visit that repository in sourcegraph.com/search. After navigating through the code base after a while, you should see the hover tooltips and definition results become more accurate.
+
+Currently, a repository will need 50 navigation events (hovers, jump to definition, find references) to trigger the auto-indexing procedure. We are continually tuning these heuristics to bring precise results to more repositories faster.
+
+
+## Product preview: code insights
+
+TODO: Felix will make a video
+
+The Sourcegraph developer team is prototyping code insights, which will help you answer questions you have about your code, including: 
+
+   * [Code smells](https://martinfowler.com/bliki/CodeSmell.html) over time
+   * Languages used at the organization
+   * Library adoption
+   * Test coverage over time
+   * Tracking a migration
+   * Security anti-patterns
+
+Code insights helps developers be more productive and do their jobs better, and brings the data we have in Sourcegraph together.
+
+We want to understand the bigger questions you have about your code base, and would love to hear from you! Please reach out at [feedback@sourcegraph.com](mailto:feedback@sourcegraph.com) to talk to us.
+
+
+## Sourcegraph developer features
+
+
+### AND/OR query improvements
```

Comment:
```suggestion
### Introducing AND and OR operators for code search
```

Or simply "Introducing operators for code search", or something like that, since this is an initial feature release.

Reply:
"AND and OR" reads a bit funny - could we do "Introducing AND/OR operators for code search?"

Reply:
Sounds good :-)